website/public/js/controllers/notificationCtrl.js

```diff
@@ -58,9 +58,11 @@ habitrpg.controller('NotificationCtrl',
 
     $rootScope.$watch('user._tmp.drop', function(after, before){
       // won't work when getting the same item twice?
-      if (after == before || !after) return;
-      $rootScope.playSound('Achievement_Unlocked');
-      if (after.type !== 'gear') {
+      if (_.isEqual(after, before) || !after) return;
+      var text, notes;
+      $rootScope.playSound('Item_Drop');
+
+      if (after.type !== 'gear' && after.type !== 'Quest') {
         var type = (after.type == 'Food') ? 'food' :
```

Comment:
IMO, once ternaries turn into double ternaries, it's time to go with something that is not a ternary.